test_scripts/RC/AUDIO_LIGHT_HMI_SETTINGS/SetInteriorVehicleData/002_Disallow_flow_by_policy_CLIMATE.lua

```diff
@@ -0,0 +1,55 @@
+---------------------------------------------------------------------------------------------------
```

Comment:
@GetmanetsIrina Why do we need test script for CLIMATE module?
It's already exists in another folder